.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: groovy
+os: linux
+dist: bionic
+arch: amd64
+sudo: required
+services: docker
+
+before_install:
+    - echo "This is the first attempt at .travis.yml for variantcalling"
+    - echo "This is an attempt to build the Dockerfile environment"
+    - docker build . --file ./Dockerfile --tag variantcaller
+    #- docker run -d --publish 8000:8080 variantcaller
+    - wget -qO- https://get.nextflow.io | bash 
+    - sudo mv nextflow /usr/local/bin
+
+script:
+    - echo $TRAVIS_BUILD_DIR
+    - ls $TRAVIS_BUILD_DIR
+    - nextflow run ./main.nf --align all --reads "CIN*R{1,2}*" --ref "./ref.fa" --fastp_min_base_quality 15 --fastp_max_percent_low_qual_base 40 --verbose -with-docker variantcaller;
+    - echo $?
```

Comment:
If a command returns a non-zero exit code, CI will fail so trying to echo the returned exit code will not produce any valuable insights.

Reply:
Echo $? also removed from travis.yml